entity/Cargo.toml

```diff
@@ -7,8 +7,15 @@ edition = "2021"
 trustify-common = { workspace = true }
 trustify-cvss = { workspace = true }
 
+async-graphql = { version = "7.0.5", features = ["uuid", "time"] }
 cpe = { workspace = true }
-sea-orm = { workspace = true, features = ["sqlx-postgres", "runtime-tokio-rustls", "macros", "with-json", "postgres-array"] }
+sea-orm = { workspace = true, features = [
```

Comment:
I would prefer keeping dependencies in a single line. I know in this case it can be hard to read. However, having set of single lines allows one to easily have IntelliJ sort all lines.

An alternative could be to pull out such dependencies out of this block and just add it later. Similar to: https://github.com/trunk-rs/trunk/blob/c160ed3ff7c98a94ab00f14acbbd198268a92525/examples/yew-tls/Cargo.toml#L13-L18

Reply:
@ctron, 

I'm happy to use either approach. 
Meanwhile this is done automatically by Rust-analyzer which I believe is now the default for many IDEs.
To me it seems to be an IntelliJ issue and the main reason for that is a very long import line, whatever would be a configured limit of 80, 120, etc number of characters, is not sustainable.